docs/first_pr.md

```diff
@@ -0,0 +1,61 @@
+# Getting started with EvalAI Issues:
+
+To contribute for CloudCV in EvalAI is a facile task. Please follow these steps in order to get started :-)
+
+**Step 1 : Fork**
+
+1. Fork the EvalAI repository from [this](https://github.com/Cloud-CV/EvalAI) link.
```

Comment:
Wher ever you are writing a link, I will prefer that instead of making `this` as the text, use from something from the link itself. Like
[this](https://github.com/Cloud-CV/EvalAI) --> [EvalAI](https://github.com/Cloud-CV/EvalAI)
[this](https://git-scm.com/docs/git-checkout) --> [git-checkout](https://git-scm.com/docs/git-checkout)